src/plugins/data_table/filter_view.js

```diff
@@ -206,35 +257,23 @@ var DataTableFilter = (function(options) {
     },
 
     _onToggleClick: function (event) {
```

Comment:
do we still need this 1 line method that just stops the event?  maybe there's another solution .. do we need a listener here at all?